k8s/generate_code.sh

```diff
@@ -37,14 +37,14 @@ if [[ ! -d "${GOPATH}/src/k8s.io/code-generator" ]]; then
 fi
 
 # https://github.com/kubernetes-sigs/controller-tools/tree/v0.7.0/cmd/controller-gen
```

Comment:
Suggest moving the changes in k8s/generate_code.sh to a new PR.

Reply:
This should not be included in this PR
This change was just to fit in my own env
Removed